src/components/dialog/Body.js

```diff
@@ -0,0 +1,29 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+
+import { Box } from '../../index';
+
+class Body extends PureComponent {
+  render() {
+    const { scrollable, children, ...rest } = this.props;
+
+    if (!scrollable) {
+      return children;
```

Comment:
Q: Just wondering, why exactly do you need to early return here then? Is it because it might just get scrollable too early (when it's not as big as you want it) that you could force it to not become scrollable? Cause I'd think those properties on Box wouldn't matter as long as it's not too big yet and it would automatically arrange everything when needed huh?